pypdf/_writer.py

```diff
@@ -1036,8 +1033,8 @@ def encrypt(
         owner_password: Optional[str] = None,
         use_128bit: bool = True,
         permissions_flag: UserAccessPermissions = ALL_DOCUMENT_PERMISSIONS,
-        user_pwd: Optional[str] = None,  # deprecated
-        owner_pwd: Optional[str] = None,  # deprecated
```

Comment:
Those two need to stay as long as we don't make a major version bump. As still many users are migrating from PyPDF2 (sometimes even PyPDF2 < 2.0.0), I want to keep the warnings for quite a while.